### PR TITLE
drop mfb/maskbits.h

### PR DESCRIPTION
**Pull request by sunweaver**
mfb is dead at X.org and we shipped one remnant file in Xserver/mfb/ (named maskbits.h).

This PR drops that file, finally. Plus some mfb/cfb/afb cleanup in Xserver/Imakefile.

### CODE REVIEW

**Comment by sunweaver** [comment]
@uli42: please review #266. Thanks!

File: nx-X11/programs/Xserver/Imakefile [hunk 1]
```diff
@@ -46,30 +46,22 @@ INSTPGMFLAGS =
...
 #define CFB4Libs PreFbLibs $(CFB4) PostFbLibs
 #define CFB8Libs PreFbLibs $(CFB8) PostFbLibs
 #define CFB16Libs PreFbLibs $(CFB16) PostFbLibs
 #define CFB24Libs PreFbLibs $(CFB24) PostFbLibs
 #define CFB32Libs PreFbLibs $(CFB32) PostFbLibs
 
-#define NoMFBCFBLibs PreFbLibs $(CFB) NoMfbPostFbLibs
-#define NoMFBCFB8Libs PreFbLibs $(CFB8) NoMfbPostFbLibs
-#define FbPostFbLibs  $(FB) NoMfbPostFbLibs
+#if !BuildXinerama
```
Comment (uli42) [inline nx-X11/programs/Xserver/Imakefile:52]: Why is that needed?

**Review by uli42 — APPROVED** [review]
Goodbye mfb ;-)

**Comment by sunweaver** [comment]
On  So 30 Okt 2016 21:59:06 CET, Ulrich Sibiller wrote:

> +#if !BuildXinerama
> 
> Why is that needed?

Compare with the previous MFB related code in Xserver/Imakefile. There  
obviously is an issue with dependency cycles which gets covered here.

Mike

## 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Fingerprint: 9BFB AEE8 6C0A A5FF BF22  0782 9AF4 6B30 2577 1B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de
